dask_planner/src/sql/logical/sort.rs

```diff
@@ -10,64 +12,61 @@ pub struct PySort {
     sort: Sort,
 }
 
-#[pymethods]
 impl PySort {
-    /// Returns a Vec of the sort expressions
-    #[pyo3(name = "getCollation")]
-    pub fn sort_expressions(&self) -> PyResult<Vec<PyExpr>> {
-        let mut sort_exprs: Vec<PyExpr> = Vec::new();
-        for expr in &self.sort.expr {
-            println!("The expr seen is {}", expr);
-            sort_exprs.push(PyExpr::from(expr.clone(), Some(self.sort.input.clone())));
-        }
-        Ok(sort_exprs)
-    }
-    #[pyo3(name = "getColumnName")]
-    pub fn get_column(&self, expr: PyExpr) -> String {
-        match &expr.expr {
-            Expr::Sort {
-                expr,
-                asc,
-                nulls_first,
-            } => {
-                println!("expr {}", expr);
-                println!("asc {}", asc);
-                println!("nulls_first {}", nulls_first);
-                match expr.as_ref() {
-                    Expr::Column(col) => {
-                        println!("Is a column {}", col);
-                        println!("col_name {}", col.name);
-                        col.name.clone()
-                    }
-                    _ => panic!("Sort Expression expr not of column type!"),
-                }
-            }
-            _ => panic!("Provided expression is not a sort expression"),
-        }
-    }
-    #[pyo3(name = "isAscending")]
-    pub fn is_ascending(&self, expr: PyExpr) -> bool {
-        match &expr.expr {
+    /// Returns if a sort expressions denotes an ascending sort
+    fn is_ascending(&self, expr: Expr) -> bool {
+        match expr {
             Expr::Sort {
                 expr: _,
                 asc,
                 nulls_first: _,
-            } => asc.clone(),
-            _ => panic!("Provided expression is not a sort expression"),
+            } => asc,
+            _ => panic!("Provided expression is not a sort epxression"),
         }
     }
-    #[pyo3(name = "isNullsFirst")]
-    pub fn is_nulls_first(&self, expr: PyExpr) -> bool {
-        match &expr.expr {
+    /// Returns if nulls should be placed first in a sort expression
+    fn is_nulls_first(&self, expr: Expr) -> bool {
```

Comment:
We could probably pass the Expr as a borrow here (`expr: &Expr`) and avoid some clones